templates/status.html

```diff
@@ -11,4 +11,29 @@
 
 {% block content %}
 {% include "partials/status_body.html" %}
+{% endblock %}
+
+{% block tail %}
+{{ super() }}
+<script src="{{ url_for_asset('d3.min.js') }}"></script>
+<script>
+	const content = {{ content.reports| tojson }}
+	function getReportTime(reports){
+		for(var id in reports){
+		  report_id = `report-${ reports[id].report_title_url }-heading .report-highlights .report-meta`;
```

Comment:
You're putting the entire URL into a CSS ID tag?  Is this common?  Also, how are you dealing with escaping special characters?  Or is it a URL ID and not an actual URL